app/register/template.hbs

```diff
@@ -0,0 +1,64 @@
+{{page-title 'Register'}}
+<Header>Register</Header>
+
+<div class='container mt-5'>
+
+  <form {{on 'submit' this.onSubmit}}>
+    <div class='mb-3 row'>
+      <label for='username' class='col-sm-2 col-form-label'>Username</label>
+
+      <div class='col-sm-10 form-floating mb-3'>
+        <input
+          type='text'
+          class='form-control'
+          id='floatingInput'
+          placeholder='name@example.com'
+          {{on 'input' this.onUsernameChange}}
```

Comment:
How about change this action to more generic, and reuse them in other fields, like:
```suggestion
          {{on 'input' (fn this.onPropertyChange 'username')}}
```

```js
// register/controller.js
@action
onPropertyChange(key, event) {
  this.model[key] = event.target.value;
}
```